core/scripts/cli/sponsor/create.js

```diff
@@ -33,20 +33,24 @@ const create = async (web3, artifacts, emp) => {
     .div(scalingFactor)
     .integerValue()
     .toString();
-  console.log("You'll need", fromWei(collateralNeeded), "ETH to borrow tokens");
+  const collateralCurrency = await ExpandedERC20.at(await emp.collateralCurrency());
+  const isWeth = await getIsWeth(web3, artifacts, collateralCurrency);
+  const collateralSymbol = await getCurrencySymbol(web3, artifacts, collateralCurrency);
+  console.log("You'll need", fromWei(collateralNeeded), isWeth ? "ETH" : collateralSymbol, "to borrow tokens");
```

Comment:
You do this ternary a lot because of the annoying ETH <> WETH. What do you think about just displaying WETH? Its more accurate to the user that their collateral is "WETH". Would make the code simpler, but perhaps makes the UX worst...

Reply:
Just displaying WETH is definitely simpler code. But we'd need to somehow message the user that they *don't* actually need WETH in their wallet, they need ETH. So that might mean there's still some custom messaging for the WETH case.

I don't love this (in fact, I dislike it). Any ideas/suggestions?

Reply:
Perhaps you could always show the symbol required as the collateral (DAI, WETH, USDC etc) and then show at the end of the sentence using the ternary operator a bracket showing that ETH will be converted to WETH. This is like @nicholaspai suggested, combined with the current implementation. This is not ideal as is ultimately the same as what you've got above but it might be explicit for the sponsor.

Reply:
Totally agree with @ptare's point that messaging is paramount here. We want to make sure it's very obvious that the user is expected to be transacting in ETH, not WETH.